drivers/SmartThings/matter-media/src/init.lua

```diff
@@ -101,8 +106,7 @@ local function media_playback_state_attr_handler(driver, device, ib, response)
     [CurrentState.PLAYING] = attr.playing(),
     [CurrentState.PAUSED] = attr.paused(),
     [CurrentState.NOT_PLAYING] = attr.stopped(),
-    -- TODO: Update to use buffering capability attribute once it is available
-    [CurrentState.BUFFERING] = attr.playing()
+    [CurrentState.BUFFERING] = attr.buffering()
```

Comment:
@ctowns is it okay to do this? I see the capability changes were done 7 months ago, so I think it should be present in all hubs that have 45.